### PR TITLE
Add more directed install instructions, fixes #60

### LINKED ISSUE #60: Add more setup detail for contributors
Need a bit more setup instructions for getting the site up and running locally.

### PR DESCRIPTION
**Pull request by emckenna**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -61,15 +61,31 @@ then that will speed the process along.
...
-```sh
-cd operationcode
-gem install bundler
-bundle
-bundle exec rake db:setup
-bundle exec rails server
-```
+## Mac OS X
+1. install homebrew ( http://brew.sh/ - package manager for OS X)  
+  * If you use macports, you might consider changing to homebrew.
```
Comment (nanoxd) [inline CONTRIBUTING.md]: Inflammatory. We don't want to push people away.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -61,15 +61,31 @@ then that will speed the process along.
...
-gem install bundler
-bundle
-bundle exec rake db:setup
-bundle exec rails server
-```
+## Mac OS X
+1. install homebrew ( http://brew.sh/ - package manager for OS X)  
+  * If you use macports, you might consider changing to homebrew. 
+  * This step is optional, but will make your life easier.
+2. install postgres if you dont have it already
```
Comment (nanoxd) [inline CONTRIBUTING.md]:
```diff
- 2. install postgres
+ 2. Install PostgreSQL
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -61,15 +61,31 @@ then that will speed the process along.
...
+  1. ```brew install postgres```
```
Comment (nanoxd) [inline CONTRIBUTING.md]: Single backticks surrounding one off commands

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -61,15 +61,31 @@ then that will speed the process along.
...
+  2. start postgres ( follow brew directions after install )
+  3. ```createuser davidmolina -d```
```
Comment (nanoxd) [inline CONTRIBUTING.md]: One must not create a user named davidmolina on other systems. The username in `config/database.yml` should be changed to ignore that for dev/test environments

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -61,15 +61,31 @@ then that will speed the process along.
...
+3. install bundler ( http://bundler.io )
```
Comment (nanoxd) [inline CONTRIBUTING.md]: Use proper markdown for links and capitalize first letter of step

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -61,15 +61,31 @@ then that will speed the process along.
...
+4. Install rbenv ( via homebrew or github, https://github.com/sstephenson/rbenv )
```
Comment (nanoxd) [inline CONTRIBUTING.md]: Same as above

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -61,15 +61,31 @@ then that will speed the process along.
...
+  1. ```brew update```
```
Comment (nanoxd) [inline CONTRIBUTING.md]: Single backticks

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -61,15 +61,31 @@ then that will speed the process along.
...
+  2. ```brew install rbenv ruby-build```
```
Comment (nanoxd) [inline CONTRIBUTING.md]: Single backpacks and we should probably not harp on what package manager users use. I myself don't use rbenv or rvm.

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -61,15 +61,31 @@ then that will speed the process along.
...
+5. clone forked operationcode repo
+6. Install Ruby 2.2.2.
+  1. ```cd <forked operationcode repo directory>```
```
Comment (nanoxd) [inline CONTRIBUTING.md]: Single backticks. Change to `cd operationcode` based on the user cloning the directory to their box in the last step

**Comment by nanoxd** [comment]
Thanks for attempting to make this better. I added a few comments for changes that should be made. 

One main point I'd like to drive home is to avoid wrangling the user into your build setup. Some may already have a ruby version manager installed and these steps will only cause issues in their dev environment. There are many tutorials out there surrounding how to install ruby; link to one of them that gives different options on that topic. 

List the ruby version, the steps required to get setup, and leave anything else out. If people have questions, we can always improve the documentation in the future :smile:

File: CONTRIBUTING.md (hunk 4 above)
Comment (nanoxd) [inline CONTRIBUTING.md]: #64 created to track that

**Comment by emckenna** [comment]
I had the mindset of someone never setting up a project before.  But I see your point.  Added the suggested changes.

**Pull request merged**